keanu-python/keanu/optimization.py

```diff
@@ -0,0 +1,62 @@
+from py4j.java_gateway import java_import
+from keanu.base import JavaObjectWrapper
+from keanu.context import KeanuContext
+from keanu.inference import BayesNet
+from keanu.vertex import Vertex
+
+k = KeanuContext().jvm_view()
+
+java_import(k, "io.improbable.keanu.algorithms.variational.optimizer.gradient.GradientOptimizer")
+java_import(k, "io.improbable.keanu.algorithms.variational.optimizer.nongradient.NonGradientOptimizer")
+
+
+class Optimizer:
+    def __init__(self, optimizer, net):
+        self.optimizer = optimizer
+        self.net = net
+
+    def max_a_posteriori(self):
+        return self.optimizer.maxAPosteriori()
+
+    def max_likelihood(self):
+        return self.optimizer.maxLikelihood()
+
+    @staticmethod
+    def _build_bayes_net(builder, net):
+        if isinstance(net, BayesNet):
+            return builder.bayesianNetwork(net.unwrap()), net
```

Comment:
this line appears twice, so it would be neater to move it to the end of the method